server/master/src/main/java/org/apache/accumulo/master/Master.java

```diff
@@ -1202,7 +1205,13 @@ public void run() {
               try {
                 TServerConnection connection = tserverSet.getConnection(server);
                 if (connection != null) {
-                  connection.halt(masterLock);
+                  // Attempt to shutdown server only if able to acquire. If unable, this table
+                  // server
```

Comment:
Just realized a spelling error and the formatting is off. I will address this tomorrow.